CHANGELOG.md

```diff
@@ -15,9 +15,9 @@
 
 ### Fixes
 
+* **Fixes category_depth None value for Title elements** Problem: The mapping of `Headline` and `Subheadline` element types from `chipper` to `Title`, assigns metadata `category_depth` 1 and 2 respectively. During the same mapping, the `Title` elements from `chipper` are left unchanged, with metadata `category_depth` = None. Fix: Whenever `Headline` or `Subheadline` type layout elements are present in a page, then all `Title` elements with `category_depth` = None should be set to have a depth of 0 instead.
```

Comment:
does this make sense semantically?
remind me what  the `category_depth` for an `H1` element is again? if it is 0, does this imply all other Titles from Chipper are like html `H1`'s?

Please add "Why does it matter? for the 2nd sentence.

Reply:
correct, `H1` is `category_depth` 0. For, `chipper`: `Title` elements are like `H1`, `Headline` like `H2`, and `Subheadline` like `H3`. I am changing the description of the fix for clarification.